modules/apps/dynamic-data-mapping/dynamic-data-mapping-service/src/main/java/com/liferay/dynamic/data/mapping/internal/util/DDMIndexerImpl.java

```diff
@@ -684,7 +684,9 @@ else if (value instanceof Object[]) {
 					document.addDate(name.concat("_date"), dateValues);
 				}
 			}
-			else if (type.equals(DDMFormFieldTypeConstants.RICH_TEXT)) {
+			else if (type.equals(DDMFormFieldTypeConstants.RICH_TEXT) ||
+					 type.equals(DDMFormFieldTypeConstants.TEXT)) {
+
```

Comment:
@adolfopa I'm not sure if calling `_htmlParser.extractText(valueString)` for text fields [here](https://github.com/liferay-content-management/liferay-portal/blob/07b31901b2131e9529ec8562f72cbfd20ad02129/modules/apps/dynamic-data-mapping/dynamic-data-mapping-service/src/main/java/com/liferay/dynamic/data/mapping/internal/util/DDMIndexerImpl.java#L696) could cause unintended consequences.

Reply:
Makes sense. I'll split this in two different branches, and treat them independently.